### ISSUE TITLE
lavaanPlot not displaying anything in R Studio

### ISSUE THREAD

**Issue by zby0327** [alishinski/lavaanPlot#49]
Hello @alishinski, 

In R Studio, I tried 

```
mod1 <- 'f1 =~ NA*wb101 + wb102 + wb103 + wb104 + wb105
          f2 =~ NA*wb201 + wb202 + wb203 + wb204 + wb205 + wb206 + wb207 + wb208 + wb209 + wb210
          f3 =~ NA*wb301 + wb302 + wb303 + wb304 + wb305 + wb306 + wb307 + wb308 + wb309
          f1 ~~ 1*f1
          f2 ~~ 1*f2
          f3 ~~ 1*f3'

fit.mod1 <- lavaan::cfa(mod1, data = dat.wb.sq, estimator = "ML")

lavaan::summary(fit.mod1, standardized=TRUE, fit.measures=TRUE)

# diagram
lavaanPlot::lavaanPlot(model = fit.mod1, 
                       node_options = list(shape = "box", fontname = "Helvetica"), 
                       edge_options = list(color = "grey"),
                       coefs = T, stand=T, covs = T)
```

but nothing was displayed. A pop-up webpage could be opened but it is blank. I also tried saving the output as pdf or png, but the output files cannot be opened by Windows. I tried different versions of the package lavaanPlot and tried the latest version on GitHub, but it did not work. 

Other visualization packages like semPlot work, so I believe this is weird that lavaanPlot is not working.
My R version is 4.4.0, and all other packages are up to date from CRAN. I also reinstalled R and Rstudio but it still does nothing. 

I appreciate it if you could figure the potential problems. Thank you.

**Comment by alishinski** [comment]
You're not the first person to ask me about this issue. 

It's caused by a known issue with the RStudio viewer with version 4.4.0: https://github.com/rstudio/rstudio/issues/14603

Good news being they patched it with version 4.4.1: https://cran.r-project.org/bin/windows/base/rpatched.html

So grab that update and let me know if that still didn't work.

**Comment by zby0327** [comment]
Thank you sooooooo much! That's perfect! You helped my whole class!
